contrib/initramfs/hooks/zfsunlock.in

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+PREREQ="dropbear"
+
+prereqs() {
+    echo "$PREREQ"
+}
+
+case "$1" in
+    prereqs)
+        prereqs
+        exit 0
+    ;;
+esac
+
+. /usr/share/initramfs-tools/hook-functions
+
+if [ -r "${DESTDIR}/sbin/dropbear" ]; then
+  cat >"$DESTDIR"/bin/zfsunlock <<'EOF'
```

Comment:
Is it reasonably possible to install the zfsunlock script directly, rather than having it be created using a big heredoc in this script?

Reply:
I've used heredoc to prevent the user to enter zfsunlock command and falsely think there are errors in the boot sequence the case user decides no longe user dropbear-initramfs package, and switch to KVM console. Some of hosting providers offer this feature. Initramfs image is rebuilt after dropbear-initramfs uninstall and zfsunlock will be completely gone.

Reply:
I don't see how that's related. If your script simply `cp`ed/`install`ed  a copy of zfsunlock, it would have the exact same effect as writing it out with a heredoc.